packages/plugins/plugins/plugin-aws/src/connections/AwsS3Bucket/AwsS3PresignedPostPolicy/AwsS3PresignedPostPolicy.js

```diff
@@ -16,14 +16,15 @@
 
 import AWS from 'aws-sdk';
 import schema from './schema.js';
+import { type } from '@lowdefy/helpers';
 
 function AwsS3PresignedPostPolicy({ request, connection }) {
   const { accessKeyId, secretAccessKey, region, bucket } = connection;
-  const { acl, conditions, expires, key } = request;
+  const { acl, conditions, expires, key, fields = {} } = request;
   const params = {
     Bucket: bucket,
     Fields: {
-      key,
+      key: key,
```

Comment:
Is this here for a specific reason? If so then I think it needs a comment so it doesn't get changed back